MatrixKit/Views/EncryptionKeys/MXKEncryptionKeysExportView.m

```diff
@@ -107,9 +108,9 @@ - (void)showInUIViewController:(UIViewController*)viewController
                                                                NSString *confirmation = textField.text;
 
                                                                // Check they are valid
-                                                               if (password.length == 0 || ![password isEqualToString:confirmation])
+                                                               if (password.length < self.passphraseMinLength || ![password isEqualToString:confirmation])
                                                                {
-                                                                   NSString *error = password.length ? [NSBundle mxk_localizedStringForKey:@"e2e_passphrase_not_match"] : [NSBundle mxk_localizedStringForKey:@"e2e_passphrase_empty"];
+                                                                   NSString *error = password.length < self.passphraseMinLength ? !password.length ? [NSBundle mxk_localizedStringForKey:@"e2e_passphrase_empty"] : [NSString stringWithFormat:[NSBundle mxk_localizedStringForKey:@"e2e_passphrase_too_short"], self.passphraseMinLength] : [NSBundle mxk_localizedStringForKey:@"e2e_passphrase_not_match"];
```

Comment:
The double usage of ternary operator is hard to read, we should probably use intermediate and/or if conditions.